.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
   - curl -o- -L https://yarnpkg.com/install.sh | bash
   - export PATH=$HOME/.yarn/bin:$PATH
 node_js:
-  - "12.13.0"
+  - '12.13.0'
```

Comment:
Hm, this is a completely different version than the one in `.tool-versions`. Aside that: does the project even still run on Travis (👴). Didn't they shut down?